src/alert/alert.style.tsx

```diff
@@ -143,3 +147,31 @@ export const TextContainer = styled.div`
     flex: 1;
     overflow-wrap: anywhere;
 `;
+/*
+ * Height is calculated based on text line height
+ * max-height: text-line-height * maximum number of lines
+ */
+
+export const TextWrapperContainer = styled.div<TextContentProps>`
+    ${(props) => {
+        const gradient =
+            "linear-gradient(to bottom, black 50%, transparent 100%)";
+        if (!props.$showMore && props.$maxHeight)
+            return `
+                height: ${props.$maxHeight}px;
+				overflow: hidden;
+                -webkit-mask-image: ${gradient};
+                mask-image: ${gradient};
+			`;
+    }}
+`;
+
+export const ShowMoreButton = styled(Text.BodySmall)`
```

Comment:
it is more semantically correct to use a div/button instead. You can use the `TextStyleHelper` if you want the `BodySmall` font